xml/System.Security.Cryptography.X509Certificates/X509ChainPolicy.xml

```diff
@@ -432,17 +430,23 @@
         <ReturnType>System.DateTime</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>The time that the certificate was verified expressed in local time.</summary>
+        <summary>Gets or sets the time for which the chain is to be validated.</summary>
         <value>A <see cref="T:System.DateTime" /> object.</value>
         <remarks>
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This property returns a value indicating the time the certificate was validated, expressed in local time. This is important when validating signed messages, since the signature must have been valid at the time of signing, not at the time of validation.  
-  
-   
-  
-## Examples  
+This property represents the time parameter for chain validation. This property returns a value indicating the time the chain is to be validated, either as local time or UTC, depending on the value of the <xref:System.DateTime.Kind?displayProperty=nameWithType> property. 
+This property is important when validating signed messages, since the signature must have been valid at the time of signing, not at the time of validation. 
+Every certificate in the chain must have a <xref:System.Security.Cryptography.X509Certificates.X509Certificate2.NotBefore> time no later than this property's value and a 
```

Comment:
Since NotBefore and NotAfter are both members of X509Certificate2 rather than X509CertificateChain, these should be ?displayProperty=nameWithType